src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -285,7 +307,7 @@ def load_duration(key: str, location: str) -> pd.DataFrame:
         [duration_draws], columns=metadata.ARTIFACT_COLUMNS, index=demography.index
     )
 
-    return duration.droplevel("location")
+    return duration
```

Comment:
there's a few dropped location levels in here, since the new demographic dimensions column doesn't have that